README.md

```diff
@@ -19,12 +21,27 @@ module.exports = {
       resolve: `gatsby-plugin-stencil`,
       options: {
         // The module of your components (required), eg "@ionic/core".
+        // Add multiple modules by using an array eg ["@ionic/core", "my-stencil-components"]
         module: "your-stencil-components-package",
 
         // Stencil renderToString options (optional): https://stenciljs.com/docs/hydrate-app#configuration-options
         renderToStringOptions: {
           prettyHtml: true,
+          removeHtmlComments: true,
+          // Must be a string as it's "evaled" by the plugin
+          afterHydrate: `(document) => document.querySelectorAll('style').forEach((tag) => tag.parentElement.removeChild(tag));`,
```

Comment:
just curious, why get the parent and removeChild and not just tag.remove()?

Reply:
Ignorance